### ISSUE TITLE
Spazzy simulator animations in XCode 6

### ISSUE THREAD

**Issue by jtmarmon** [zrxq/UIView-EasingFunctions#15]
Using the easing functions on my iPhone 5s is fine, but when I move to the 5s simulator in XCode 6, the animations are super spazzy.

**Comment by dannybabiy** [comment]
Have you figured out a fix/workaround ?

**Comment by jtmarmon** [comment]
Nope! Didn't really need one since I can just debug on my phone although it is slightly irritating

**Comment by glenstorey** [comment]
Yeah also having this problem - this issue saved me from rolling back to xCode 5 though! I thought the simulator was gospel.

**Comment by tapthaker** [comment]
Specifically created a new repo which changes the implementation inside but works almost in the same way, couldn't add bouncy & elastic animations but if you are using other easings you should try it. https://github.com/thegreatloser/UIView-Easing
